OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -128,9 +127,7 @@ List<CPos> CalculatePathToTarget(Actor self, BlockedByActor check)
 			if (!searchCells.Any())
 				return PathFinder.NoPath;
 
-			using (var fromSrc = PathSearch.ToTargetCell(self.World, Mobile.Locomotor, self, searchCells, loc, check))
-			using (var fromDest = PathSearch.ToTargetCell(self.World, Mobile.Locomotor, self, loc, lastVisibleTargetLocation, check, inReverse: true))
-				return Mobile.Pathfinder.FindBidiPath(fromSrc, fromDest);
+			return Mobile.PathFinder.FindUnitPathToTargetCell(self, searchCells, loc, check);
```

Comment:
I'd guess the regression is related to this change, which seems to be more complex than the other changes.

Reply:
Excellent analysis, you are correct.

Thankfully the issue here was straightforward, I have the source and target locations swapped compared to the original! The obvious fix of swapping them doesn't work because multiple cells are only supported from the source; there can be only one target location. Instead, we perform the same search but reverse the returned path.